helm/templates/servicemanager/jmx-configmap.yaml

```diff
@@ -0,0 +1,327 @@
+{{- if .Values.servicemanager.enabled }}
```

Comment:
is this needed for single node setup?

Reply:
May be not, but I copied over the rules in case at some point we need to enable the metrics reporting to debug something. We can have it disabled by default

I missed having `.Values.servicemanager.jmx.enabled` in condition. Fixed it now